components/dashboard/src/projects/Projects.tsx

```diff
@@ -90,23 +97,16 @@ export default function () {
                         <div className="py-4">
                             <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16" width="16" height="16"><path fill="#A8A29E" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" /></svg>
                         </div>
-                        <input type="search" placeholder="Search Projects" onChange={(e) => onSearchProjects(e.target.value)} />
+                        <input type="search" placeholder="Search Projects" onChange={e => setSearchFilter(e.target.value)} />
                     </div>
                     <div className="flex-1" />
                     <div className="py-3 pl-3">
-                        <DropDown prefix="Status: " contextMenuWidth="w-32" activeEntry={'Recent'} entries={[{
-                            title: 'Recent',
-                            onClick: () => { /* TODO */ }
-                        }, {
-                            title: 'All',
-                            onClick: () => { /* TODO */ }
-                        }]} />
                     </div>
                     <Link to="./members" className="flex"><button className="ml-2 secondary">Invite Members</button></Link>
                     <button className="ml-2" onClick={() => onNewProject()}>New Project</button>
                 </div>
                 <div className="mt-4 grid grid-cols-3 gap-4">
-                    {projects.map(p => (<div key={`project-${p.id}`} className="h-48">
+                    {projects.filter(filter).map(p => (<div key={`project-${p.id}`} className="h-48">
```

Comment:
**suggestion**: Could we hide the New Project card when there's no search result? Ideally, a minimal empty state like the one in https://github.com/gitpod-io/gitpod/issues/5022 with a link to create a new project could suffice. Sounds good also as a follow up issue. 🏓 

<img width="1105" alt="Screenshot 2021-08-23 at 3 34 52 PM" src="https://user-images.githubusercontent.com/120486/130447916-1cbabaa9-8867-4148-aec0-74796e191599.png">

Reply:
not showing the `New Project` card on active search sounds great in general.

Reply:
**fyi**: Added follow up issue in https://github.com/gitpod-io/gitpod/issues/5343.